internal/restic/node.go

```diff
@@ -225,6 +270,24 @@ func (node Node) restoreMetadata(path string) error {
 		}
 	}
 
+	if err := node.restoreGenericAttributes(path); err != nil {
+		debug.Log("error restoring generic attributes for %v: %v", path, err)
+		if firsterr != nil {
+			firsterr = err
+		}
+	}
+
+	//Moving RestoreTimestamps and restoreExtendedAttributes calls above as for readonly files
```

Comment:
Added space after `//`.
Regarding the ordering for readonly which only applies to windows, I guess it's fine to change the order in the common code (as currently implemented) as it would not affect the functionality for other OSs. Let me know if you disagree.